src/store/hotspots/hotspotsSlice.ts

```diff
@@ -323,7 +323,11 @@ export const fetchSyncStatus = createAsyncThunk(
     }
 
     if (!statusTime) {
-      return { status: SyncStatus.none, percent: 0 }
+      return {
+        status: SyncStatus.none,
+        percent: 0,
+        hotspotBlockHeight: 0,
```

Comment:
I'm not sure it matters too much, but on the explorer I've been using height = 1 for no sync because the miner firmware will come with a genesis block onboard, so it starts syncing from block 1

Reply:
Cool. I updated it.